ext/civicrm_admin_ui/ang/afsearchRelationshipTypes.aff.html

```diff
@@ -0,0 +1,13 @@
+<div class="af-container">
+  <div class="af-markup">
+    <div class="help">
+    <p>{{:: ts(&#39;Relationship types describe relationships between people, households and organizations. Relationship types labels describe the relationship from the perspective of each of the two entities (e.g. Parent &gt;-&lt; Child, Employer &gt;-&lt; Employee). For some types of relationships, the labels may be the same in both directions (e.g. Spouse &gt;-&lt; Spouse).&#39;) }} <a class="crm-doc-link no-popup" href="https://docs.civicrm.org/user/en/latest/organising-your-data/relationships" target="_blank">{{:: ts(&#39;(Learn more...)&#39;) }}</a></p>
+
+    <p>{{:: ts(&#39;You can define as many additional relationships types as needed to cover the types of relationships you want to track. Once a relationship type is created, you may also define custom fields to extend relationship information for that type from &#39;) }} <a href="civicrm/admin/custom/group?reset=1">{{:: ts(&#39;Administer CiviCRM &raquo; Custom Data&#39;) }}</a></p>
```

Comment:
I don't think escaping straight-quotes with `&#39;` is necessary, and it may prevent i18n string parsing

Reply:
True. This was auto-generated during creation of the form, I've replaced it in the new commit.